README.md

```diff
@@ -1,2 +1,36 @@
-# beyond-os-06-group-3
-Project repository for beyond-OS 06 Group #3
+![Logo](https://user-images.githubusercontent.com/92451459/139893293-8fae2878-5cb8-474e-8b0c-a98a1d052842.jpg)
+
+
+# Student JMaster 
+
+Student jobs app
+
+# Description
+A platform for students to find jobs/internships.
+The website allows a student to find a job or an internship that is relevant to their major.
+The jobs can be organized according to a few filters, such as experince needed and location.
```

Comment:
experience?

Reply:
work experience required for the job if needed

Reply:
yes I meant you had a typo in that word

Reply:
My bad 
Will fix it now